### PR TITLE
Add onScreenActivated event

### PR DESCRIPTION
**Pull request by schmave**

### CODE REVIEW

File: tests/image_gen/cs3_screens.py [hunk 1]
```diff
@@ -1,5 +1,20 @@
+calledStart = False
+calledActivated = False
+
+def a_onScreenStart(app):
```
Comment (schmave) [inline tests/image_gen/cs3_screens.py:4]: Does this test capture the requirements for onScreenActivated? All screens' onScreenStart functions are called when runAppWithScreens is called, then activeScreen_onScreenActivated is called, and then activeScreen_redrawAll. 

After that, newScreen_onScreenActivated is called whenever setActiveScreen('newScreen') is called.

**Comment by dkosbie** [comment]
This seems very reasonable to me.  Thx.

**Comment by schmave** [comment]
@lsands50 @dkosbie This is now available on pip and in the zip download.

**Comment by dkosbie** [comment]
Awesome!  Thanks!  :-)

**Pull request merged**